src/azul/__init__.py

```diff
@@ -162,6 +162,13 @@ def dss_direct_access(self) -> bool:
     def dss_direct_access_role(self, lambda_name: str) -> Optional[str]:
         return os.environ.get('AZUL_DSS_DIRECT_ACCESS_ROLE').format(lambda_name=lambda_name)
 
+    @property
+    def dss_direct_access_account(self) -> Optional[str]:
+        """
+        The direct access account id extracted from the direct access role value
+        """
+        return os.environ.get('AZUL_DSS_DIRECT_ACCESS_ROLE').split(':')[4]
```

Comment:
What if `.get()` returns `None`?

Reply:
@jessebrennan, this one is easy to catch, wouldn't you agree?

Reply:
The env variable `AZUL_DSS_DIRECT_ACCESS_ROLE` is already expected to be required by `dss_direct_access_role()`.
I agree however that if a key is required then `[]` should be used not `.get()`. I will update along with `dss_direct_access_role()` to match.

Reply:
Yes, it seems that I made the same mistake in config.dss_direct_access_role.

Reply:
Yes, I should have seen this.